### DISCUSSION TITLE
[Feature request] Minimize search bar location

### DISCUSSION THREAD

**Discussion by quartertone** [standardnotes/forum#2527] [Suggestions] [answered]
Currently (Android SN v3.0.3) the search bar exists at the very top of the list of notes. This creates a tedious problem sometimes, like in this scenario:

After having scrolled down a lot in the note list, I decide I want to do a search rather than visually looking for the note I'm interested in. So now I have to scroll all the way up to the top of the list of notes to get to the search bar before I can enter my search.

This problem is compounded by the issue where the note list does not follow the usual "fling" behavior, so the scrolling has to be done manually one screenful at a time.

It would be more convenient to have the search bar persistently at the top of the display, or hidden in the side panel.

**Comment by atmoio** [comment]
Noted. On iOS this isn't as much a problem since you can top the status bar to immediately scroll to the top of the list. But on Android I believe the usual paradigm is a search button in the nav bar that toggles down a search bar?

**Comment by ChristianKrebel** [comment]
> But on Android I believe the usual paradigm is a search button in the nav bar that toggles down a search bar?

No, a search button in the action bar would be the usual and it would also save space.

![grafik](https://user-images.githubusercontent.com/22580809/54503062-db1d5780-492d-11e9-9923-bfb74e234cd7.png)

**Comment by dardoor** [comment]
I vote for anything that moves search out of the way. I don't use it and it's just taking up space where it is now.

**Comment by myreli** [comment, marked as answer]
The [new mobile experience](https://standardnotes.com/blog/our-new-mobile-experience-is-launching-soon) pins the search bar even when scrolling through the notes. Let us know if you face any issues. 

<img width="484" alt="image" src="https://user-images.githubusercontent.com/17554234/218593093-d1f3897e-b275-4d65-8bf1-eabbd31ffe3e.png">
